### DISCUSSION TITLE
USB pass through to Docker possible?

### DISCUSSION THREAD

**Discussion by FlaMike** [tteck/Proxmox#483] [Q&A] [answered]
I just installed Docker on Proxmox using the Proxmox helper script for Home Automation.  The script installs Docker in an unprivileged container so it's not possible when installed to pass through USB devices.  The Proxmox dashboard does not permit me to change it to a privileged container so I may pass through a USB device.  Is there another way to change the container so I can pass through a USB device?  Thanks!

**Comment by tteck** [comment, marked as answer]
Have you tried to restore from a backup and change it to privileged? 

![Screenshot 2022-09-05 4 37 29 PM](https://user-images.githubusercontent.com/21342670/188510712-0fae590d-448c-4fb0-a8b5-5820b7f4ec5d.png)

It'll create a new CTID

**Reply by FlaMike** [reply]
I have not tried that approach--I literally just ran the script.  Could I just create an (empty) backup & then restore it with the change?

Thanks for your quick response!!!

**Reply by tteck** [reply]
When you run the script select NO to using default settings and you can change it there

**Comment by FlaMike** [comment]
Will do.  Thank you!!!!
